src/routes/safe/components/Apps/components/AppFrame.tsx

```diff
@@ -74,6 +92,7 @@ const AppFrame = forwardRef<HTMLIFrameElement, AppFrameProps>(function AppFrameC
     <IframeWrapper>
       {appIsLoading && (
         <LoadingContainer>
+          {appTimeout && <Title size="xs">The IPFS provider is slow, the app might not load properly</Title>}
```

Comment:
(A user) What is IPFS provider? 😛 

Maybe we can think about a less technical description

Reply:
Yeah you are right, maybe something like `We are not being able to connect with the app, it might not load properly.`?
Or something else, do you have something in mind?

Reply:
We are not being able to load the safe-app.